meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -230,7 +230,7 @@ MENDER_DATA_PART_LABEL ??= "data"
 # --------------------------- END OF CONFIGURATION -----------------------------
 
 IMAGE_INSTALL:append = " mender-client"
-IMAGE_CLASSES += "mender-part-images mender-ubimg mender-artifactimg mender-dataimg mender-bootimg mender-datatar"
+IMAGE_CLASSES += "mender-part-images mender-ubimg mender-artifactimg mender-artifactcapsule mender-dataimg mender-bootimg mender-datatar"
```

Comment:
Even though the original is concatenated, it's better to split up the name I think. I suggest `mender-artifact-uefi-capsule` here as well.

Reply:
done

Reply:
I meant the name as well, rename from `mender-artifactcapsule` to `mender-artifact-uefi-capsule`.

Reply:
Done